src/ConfigContext.ts

```diff
@@ -42,6 +42,9 @@ export interface ConfigContext {
   ARM_API_VERSION: string;
   GRAPH_ENDPOINT: string;
   GRAPH_API_VERSION: string;
+  CATALOG_ENDPOINT: string;
```

Comment:
Please add a comment about what this endpoint is

Reply:
@jawelton74 Comment added, thank you for pointing this out!